packages/dev/query/src/index.js

```diff
@@ -120,9 +124,9 @@ export function loadGraphs(cacheDir: string): {|
   return {assetGraph, bundleGraph, requestTracker, bundleInfo};
 }
 
-function loadLargeBlobRequestRequestSync(cacheDir, node) {
+async function loadLargeBlobRequestRequestSync(cache, node) {
```

Comment:
I'm not sure why this was originally written to be sync, but if we're making it async then we should rename the function.

Reply:
I think I made `loadGraphs` sync to make it easier to use from the Node REPL